src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerAdminPostAproveDeletionIntTest.java

```diff
@@ -0,0 +1,168 @@
+                                                .build());
+        // run the test
+        MvcResult mvcResult = mockMvc.perform(post(ENDPOINT_URL.replace(
+                "${TRANSACTION_DOCUMENT_ID}", transcriptionDocumentEntity.getId().toString()))
+                                                  .header("Content-Type", "application/json"))
+            .andExpect(status().is2xxSuccessful())
+            .andReturn();
+
+        TranscriptionDocumentEntity documentEntity = transcriptionDocumentRepository.findById(transcriptionDocumentEntity.getId()).get();
+        List<ObjectAdminActionEntity> objectAdminActionEntity = objectAdminActionRepository.findByTranscriptionDocument_Id(transcriptionDocumentEntity.getId());
+
+        TranscriptionDocumentHideResponse transcriptionResponse
+            = objectMapper.readValue(mvcResult.getResponse().getContentAsByteArray(), TranscriptionDocumentHideResponse.class);
+
+        // ensure that the database data is contained in the response
+        assertEquals(documentEntity.getId(), transcriptionResponse.getId());
+        assertEquals(documentEntity.isHidden(), transcriptionResponse.getIsHidden());
+        assertEquals(objectAdminActionEntity.getFirst().getId(), transcriptionResponse.getAdminAction().getId());
+        assertEquals(objectAdminActionEntity.getFirst().getComments(), transcriptionResponse.getAdminAction().getComments());
+        assertEquals(objectAdminActionEntity.getFirst().getTicketReference(), transcriptionResponse.getAdminAction().getTicketReference());
+        assertEquals(objectAdminActionEntity.getFirst().getObjectHiddenReason().getId(), transcriptionResponse.getAdminAction().getReasonId());
+        assertTrue(objectAdminActionEntity.getFirst().isMarkedForManualDeletion());
+        assertEquals(objectAdminActionEntity.getFirst().getHiddenBy().getId(), transcriptionResponse.getAdminAction().getHiddenById());
+        assertEquals(objectAdminActionEntity.getFirst()
+                         .getHiddenDateTime().truncatedTo(ChronoUnit.SECONDS),
+                     transcriptionResponse.getAdminAction().getHiddenAt().truncatedTo(ChronoUnit.SECONDS));
+        assertEquals(objectAdminActionEntity.getFirst().getMarkedForManualDelBy().getId(),
+                     transcriptionResponse.getAdminAction().getMarkedForManualDeletionById());
+        assertEquals(objectAdminActionEntity.getFirst().getMarkedForManualDelDateTime()
+                         .truncatedTo(ChronoUnit.SECONDS), transcriptionResponse.getAdminAction()
+                         .getMarkedForManualDeletionAt().truncatedTo(ChronoUnit.SECONDS));
+    }
+
+    @Test
+    void testTranscriptionDocumentApproveDeletionForbidden() throws Exception {
+        superAdminUserStub.givenUserIsAuthorised(userIdentity, SecurityRoleEnum.SUPER_USER);
+
+        mockMvc.perform(post(ENDPOINT_URL.replace(
+                "${TRANSACTION_DOCUMENT_ID}", Integer.valueOf(-12).toString()))
```

Comment:
ENDPOINT_URL.replace(XXX,YYY) can be extracted into a private method to save repeating the same logic :)

I.e 
```

private String getUrl(String transcationDocumentId) {
  return ENDPOINT_URL.replace("${TRANSACTION_DOCUMENT_ID}",transcationDocumentId);
} 

then use post(getUrl("-12"))
```

Reply:
Done